docs/source/user-guide/using-virelay-with-gunicorn.rst

```diff
@@ -0,0 +1,20 @@
+===========================
+Using ViRelAy With Gunicorn
+===========================
+
+As described in the :doc:`../getting-started/basic-usage` article, ViRelAy provides a command line interface, to start the using a simple built-in HTTP server. This HTTP server is, however, rather slow, because it is limited to responding to requests in sequence. To unlock the full potential of ViRelAy or even run it on a remote server to collaborate with others, it is recommended to run ViRelAy using the WSGI HTTP server `Gunicorn <https://gunicorn.org/>`_. Gunicorn was installed together with ViRelAy, but if you want to use a specific version, it can also be installed from PyPi: ``pip install gunicorn``.
+
+To start ViRelAy using Gunicorn, you can use the ``gunicorn`` command and tell it how to create a ViRelAy application, which can be done like so:
+
+.. code-block:: console
+
+    $ gunicorn -w 4 -b 127.0.0.1:8080 "virelay.application:create_app(projects=['path/to/project-1.yaml', 'path/to/project-2.yaml'])"
+
+With the above command, the server binds to 127.0.0.1 on port 8080 and uses 4 worker processes. ViRelAy will load the two projects ``project-1.yaml`` and ``project-2.yaml``. You can specify as many projects as you like. Alternatively, the environment variable ``VIRELAY_PROJECTS`` may be used to specify the project paths as a colon-separated list like so:
```

Comment:
Although the additional tabs will not be clickable in the interface anymore (as far as I remember when I tried this out, not scroll-bar appeared)

Reply:
I don't quite understand what you mean?

Reply:
I was talking about the "as many projects as you like".
After the vertical space of project tabs is full, you cannot visually see the rest of the projects because no scrollbar will appear.

Reply:
I think we can keep the formulation this way and just fix it (somehow) later.

Reply:
Ah, okay, I understand. Maybe we should add a horizontal scrollbar then. But I think this should be done in a future release.